src/main.cpp

```diff
@@ -7,7 +7,7 @@ int main(int argc, char *argv[]){
     // set a limit of the number of arguments to be passed: in argc
     if(argc > 4){
         // tell the user how to run the program
-        std::cout<<"Usage1: passgen -L length_of_the_password_you_want\n";
+        std::cout<<"Use 'passgen -H' to print the instructions on how to use this tool\n";
```

Comment:
It is better to print usage information here instead of showing how to get usage information.